mcs/class/System/System/Uri.cs

```diff
@@ -170,9 +170,16 @@ protected Uri (SerializationInfo serializationInfo, StreamingContext streamingCo
 			}
 		}
 
+		// When used instead of UriKind.RelativeOrAbsolute paths such as "/foo" are assumed relative.
+		const UriKind DotNetRelativeOrAbsolute = (UriKind) 4;
```

Comment:
Use different number than 4 which is likely to collide if enum was extended by .net framework